ern-container-gen-android/src/hull/build.gradle

```diff
@@ -5,6 +5,9 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:{{{androidGradlePlugin}}}'
+        {{#isKotlinEnabled}}
```

Comment:
Minor: Move template conditionals (line 8 and 10) to column 1, as indentation is not part of the line-by-line evaluation. This is to avoid potential trailing spaces in the final file. (see line 16/19 etc).